GVRf/Framework/backend_daydream/build.gradle

```diff
@@ -124,3 +124,7 @@ assembleDebug {}.doLast {
         uploadToGitHub.execute();
     }
 }
+
+if(file("../release.gradle").exists()) {

```

Comment:
Please apply only in case artifactory_username is defined. Let's not force everyone define artifactory_username - dummy values or not.